global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "8.0.403"
+        "version": "9.0.102"
```

Comment:
Do we want to allow any rollforward to minor versions? Or keep it as is to a specific version?

Reply:
Mikael had a different PR trying to address this - #4687. This PR is just adding the new target